data/wanderers middle.txt

```diff
@@ -1763,7 +1763,7 @@ mission "Wanderers: Sestor Search: Varu K'prai"
 	on offer
 		conversation
 			`The entire spaceport on <planet> has been razed to the ground. Teams of Wanderers are sifting through the rubble to recover the bodies of those who were killed here. Apparently the Kor Sestor fleet destroyed the spaceport and every Wanderer ship it found in this system, and then occupied the system for several days, destroying or driving away any scout ships that entered the system.`
-			`	The Wanderers don't know what happened next, but the drones are no longer here, so they must have moved on to a different system.`
+			`	The Wanderers don't know what happened next, but the drones are no longer here, so they must have moved beyond Wanderer space.`
```

Comment:
It's a good idea. Just one small thing:
I think concluding that they "must have" moved outside of Wanderer space is a little bold. Maybe it could be written as, `the drones are no longer here. Perhaps they have moved beyond Wanderer space.` or something less conclusive as "must have moved beyond Wanderer space". I don't think you could deduce that instantly from the situation.

But yes, this would help.